src/service/ampdoc-impl.js

```diff
@@ -120,6 +120,14 @@ export class AmpDocService {
         }
       }
 
+      // Traverse the boundary of a friendly iframe.
+      const win = (n.ownerDocument || n).defaultView;
+      if (win && win != this.win && win.frameElement) {
```

Comment:
It's `null`. But also I assume that the only way I'd get access to this element is in the same-origin case.